### DISCUSSION TITLE
Explain to a noob

### DISCUSSION THREAD

**Discussion by Rongusha** [Vedza/NitroSniperGo#236] [Q&A]
Hi, im not a Python dev and i dont know if that matters lol. First time ever hear about sniper apps and yeah.. Would someone explain how to use this and how it works? I want to get my hand on nitro but nothing here makes sense for me :( Thanks

**Comment by Vedza** [comment]
Hi, It's not in python but in Go. But you don't need to know anything about coding to use it don't worry.

The easiest way to use it is to create an account on Heroku, then [click here](https://heroku.com/deploy?template=https://github.com/Vedza/NitroSniperGo/tree/heroku).
Edit the settings as you want, you can [look here](https://github.com/Vedza/NitroSniperGo#usage) to have more information on each setting. You'll need to put your discord account token in it, [here is a detailed tutorial on how to get it](https://github.com/Tyrrrz/DiscordChatExporter/wiki/Obtaining-Token-and-Channel-IDs#how-to-get-a-user-token).

After you're done setting up your sniper :
*  Click on **Deploy app** at the bottom of Heroku page.
*  Go to **Resources** and enable sniper after clicking on the pencil next to it.
*  You can see logs to check if it's working in **More** *(Top right)* -> **View logs**

If you see `[+] Sniper is ready` then it's working for free 24/7 !

You can disable it the same way you enable it.

If you want to edit the settings or if there is an update, I suggest you to delete in **Settings** -> **Delete App** *(bottom)*, and start again from the beginning.

**Reply by Rongusha** [reply]
Nice, thanks i will get into it. From i see in the discussions its better to create an alt acc on discord so to do job in case it get banned right? Also is the default settings best not to get banned and if not would you recommend some settings. Thanks

**Reply by Vedza** [reply]
Actually it's quite the opposite.

Alts will increase the chance of getting Nitro, but it will make way more requests for that, so you'll get more chance of getting banned.

If you just want Nitro, I would disable `giveaway` and `invite` snipers, and just put your main token, if you have alts you can also add them, but not too many. 

Then wait a few days, if you don't get anything you can enable  `invite`, so your alts will join a lot of servers. Only alts will joins new servers, and if your alts accounts are new they will probably get banned.

And if you still don't get anything then enable `giveaway` sniper, you can leave the settings unchanged or add words in `whitelist_words` if you want to win other things, and maybe edit custom message sent if won.

But be careful because if you won giveaways people can find out you're using a sniper and report you. Your chance of getting banned are low, but it can still happen.

**Reply by Rongusha** [reply]
Great its all set up and running. I will now wait and see where i get with 1 alt and disabled giveaways and invite.Thanks a lot Great responce and hopefully this will help other noobs like me to get to it 🥇 💯

**Reply by Zorral** [reply]
> Actually it's quite the opposite.
> 
> Alts will increase the chance of getting Nitro, but it will make way more requests for that, so you'll get more chance of getting banned.
> 
> If you just want Nitro, I would disable `giveaway` and `invite` snipers, and just put your main token, if you have alts you can also add them, but not too many.
> 
> Then wait a few days, if you don't get anything you can enable `invite`, so your alts will join a lot of servers. Only alts will joins new servers, and if your alts accounts are new they will probably get banned.
> 
> And if you still don't get anything then enable `giveaway` sniper, you can leave the settings unchanged or add words in `whitelist_words` if you want to win other things, and maybe edit custom message sent if won.
> 
> But be careful because if you won giveaways people can find out you're using a sniper and report you. Your chance of getting banned are low, but it can still happen.

Btw, do you have to enter in the invites yourself or does it join random servers with the whitelisted words?

**Reply by Vedza** [reply]
It joins every giveaway with white listed word in it automatically and join automatically invite links following rules (max, delay, cooldown), there isn't any whitelist for invites.

**Reply by Zorral** [reply]
ok, but as a suggestion, you can try to make it join random servers with tags like nitro gaw, and such if it's not too hard. And I was also wondering why it only works with some bots.

**Reply by Vedza** [reply]
It only works with some bots because it detect giveaway with `React with..` and/or `giveaway` in it, and then react with 🎉, and some bot send other messages and works with other emote.
About random servers I don't know where can I find server list with nitro tags that are not full of bots.

**Reply by Zorral** [reply]
> It only works with some bots because it detect giveaway with `React with..` and/or `giveaway` in it, and then react with 🎉, and some bot send other messages and works with other emote.
> About random servers I don't know where can I find server list with nitro tags that are not full of bots.

I believe u can go straight to discord -> explore servers -> search nitro (this gives u all the servers with tags of nitro), don't know if that's possible in go but if u can, will definitely be nice!

**Reply by Zorral** [reply]
> Actually it's quite the opposite.
> 
> Alts will increase the chance of getting Nitro, but it will make way more requests for that, so you'll get more chance of getting banned.
> 
> If you just want Nitro, I would disable `giveaway` and `invite` snipers, and just put your main token, if you have alts you can also add them, but not too many.
> 
> Then wait a few days, if you don't get anything you can enable `invite`, so your alts will join a lot of servers. Only alts will joins new servers, and if your alts accounts are new they will probably get banned.
> 
> And if you still don't get anything then enable `giveaway` sniper, you can leave the settings unchanged or add words in `whitelist_words` if you want to win other things, and maybe edit custom message sent if won.
> 
> But be careful because if you won giveaways people can find out you're using a sniper and report you. Your chance of getting banned are low, but it can still happen.

Also as a side note, Noumenon is a very popular bot that a lot of servers use but it doesn't work with this code (think it was made to prevent this from happening lol). Hopefully u can get to checking it out if u have some free time!

**Reply by Zorral** [reply]
Here are some information about Noumenon:
Bot Tag: Noumenon#7110
Bot ID: 634866217764651009
Invite Link: https://discord.com/oauth2/authorize?client_id=634866217764651009&permissions=2080894207&scope=bot+applications.commands
Default Prefix & Giveaway Start Command: !g start
